spec.bs

```diff
@@ -1933,33 +2056,39 @@ SKOS constructs such as <code>skos:narrower</code> <em class="rfc2119">MAY</em>
 Definitions {#definitions}
 =====================
 The following terms are used throughout this specification:
+  * <dfn>Agent</dfn> -- a distinct individual, group, organization, or piece of
```

Comment:
In this context, and as it relates to #14, I think that we should avoid talking about agents in an authenticated ecosystem context, and instead focus on application interoperability between client or server side agents (using agent in the context of software). otherwise there are tons of client/server-side agent references throughout the doc that become a little bit confusing. Maybe update the reference in #14 to just talk about client / server-side agents, and then put definitions for them in here instead.

Reply:
Agreed.

I removed the line within the ecosystem section that introduced this confusion as it wasn't especially meaningful anyways.  And then removed the Agent definition.  Then added client and server side definitions and link to them when used.